nx/include/switch/services/applet.h

```diff
@@ -71,6 +71,11 @@ Result appletSetScreenShotPermission(s32 val);
 
 Result appletSetScreenShotImageOrientation(s32 val);
 
+/**
+ * @brief Exits the application.
```

Comment:
*the current applet

(Assuming AppletType doesn't matter?)

Reply:
What AppletType(s) was this tested with?

Reply:
I've only tested it with an Application. Switchbrew doesn't say either way, I'll check SwIPC.

Reply:
SwIPC doesn't say either. I don't know how to build an Applet, though.

Reply:
Actually, I'll see what it does in hbmenu. That runs as an applet, right?

Reply:
It works with an application, but crashes the system with code 2144-0001 in hbmenu. I don't know how to test anything else.

Reply:
Add a comment regarding that this can be only used with AppletType_Application.

Reply:
Okay. Should I squash?

Reply:
doesn't matter